src/temporal/t.rast.aggregate.seasons/t.rast.aggregate.seasons.html

```diff
@@ -0,0 +1,43 @@
+<h2>DESCRIPTION</h2>
+<em><b>t.rast.aggregate.seasons</b></em> aggregates an input space time
+raster dataset at astronomical seasons level using a statistical method selected by the user.
+ It uses <a href="https://grass.osgeo.org/grass-stable/manuals/t.rast.aggregate.ds.html">t.rast.aggregate.ds</a>
+to detect and copy the astronomical seasons granularity.
+Astronomical seasons are defined as:
+<ul>
+    <li><em>Spring</em> 20 March - 21 June</li>
+    <li><em>Summer</em> 21 June - 20 September</li>
+    <li><em>Autumn</em> 20 September - 21 December</li>
+    <li><em>Winter</em> 21 December - 20 March (following year)</li>
+</ul>
+<p>
+
+Using the <em>output</em> option it will create a unified space time raster dataset with the name set in this option,
+otherwise it will create a space time raster dataset for each year in the input space time raster dataset.
+
+<h2>EXAMPLES</h2>
+Calculate seasonal data from an input space time raster dataset with a unified output space time raster dataset
+<div class="code"><pre>
+    t.rast.aggregate.seasons input=mystrds basename=mystrds_seasons output=outstrds
+</pre></div>
+
+Calculate seasonal data from an input space time raster dataset with yearly output space time raster datasets
+<div class="code"><pre>
+    t.rast.aggregate.seasons input=mystrds basename=mystrds_seasons
+</pre></div>
+
+<h2>NOTES</h2>
+If one of the season is not fully covered by the input space time raster dataset that season will not be created by the module.
+For example if you have a daily space time raster dataset for just one year it will create only raster maps for 3 seasons (Spring, Summer, Autumn),
+the Winter one will be avoided since your input space time raster dataset as data only until the end of the year but it miss the data
+from January to March.
```

Comment:
```suggestion
If one of the season is not fully covered by the input space time
raster dataset that season will not be created by the module. For
example if you have a daily space time raster dataset for just one year
it will create only raster maps for 3 seasons (Spring, Summer, Autumn),
the Winter one will be avoided since your input space time raster
dataset as data only until the end of the year but it miss the data
from January to March.
```

(breaking long lines)